sf-docker/docker-compose.yml

```diff
@@ -316,7 +316,19 @@ services:
       - RabbitMq__Port=${RabbitMQ__Port}
       - Kestrel__EndPoints__Http__Url=${Hosting__Host}:${Hosting__Port}
 
-  # Add smartface station when dockerized
+  sf-station:
+    image: ${REGISTRY}sf-station:${SFS_VERSION}
+    container_name: SFStation
+    restart: unless-stopped
+    ports:
+      - 80:80
+    env_file: .env.sfstation
+    environment:
+      # for now we disable keycloak-integration
+      DISABLE_AUTHENTICATION: 'true'
```

Comment:
The env variable `DISABLE_AUTHENTICATION` is not used anywhere in the code - only `ENABLE_AUTHENTICATION` is referenced. We should remove this line